src/trajectory/CMakeLists.txt

```diff
@@ -9,6 +9,7 @@ add_library("${PROJECT_NAME}_trajectory" SHARED ${sources})
 target_link_libraries("${PROJECT_NAME}_trajectory"
   PUBLIC
     "${PROJECT_NAME}_common"
+    "${PROJECT_NAME}_distance"
```

Comment:
If we keep this dependency, we should make sure to update the comments of `src/CMakeLists.txt` too.